src/App.tsx

```diff
@@ -1,27 +1,82 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+enum Sort {
+  All = 'all-button',
+  FirstFive = 'first-five-button',
+  Red = 'red-button',
+}
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+const getButtonName = (sortType: Sort) => {
```

Comment:
Nice idea to use switch/case, but I have suggestion how to make it a bit simpler
We can just create an object with enum values as keys and content strings as values and then just read it using enum value

```
const contentBySortType = {
  [Sort.All]: 'Load all goods',
  // content
}

const sortBy = Sort.All;

const content = contentBySortType[sortBy]; // 'Load all goods'
```